render-ws-spark-client/src/test/java/org/janelia/render/client/spark/pipeline/AlignmentPipelineParametersTest.java

```diff
@@ -0,0 +1,39 @@
+package org.janelia.render.client.spark.pipeline;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.janelia.render.client.newsolver.setup.AffineBlockSolverSetup;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Tests the {@link AlignmentPipelineParameters} class.
+ *
+ * @author Eric Trautman
+ */
+public class AlignmentPipelineParametersTest {
+
+    @Test
+    public void testJson()
+            throws IOException {
+        final AlignmentPipelineParameters pipelineParameters =
+                AlignmentPipelineParameters.fromJsonFile("src/test/resources/pipeline/msem_alignment_pipeline.json");
+
+        Assert.assertNotNull("deserialized parameters are null", pipelineParameters);
```

Comment:
We should consider switching to JUnit5 in the long run: it always confuses me if I read something like "assertNotNull(".. are null")". JUnit5 places the `msg` parameter at the end of the call.